src/resources/fhirReader.js

```diff
@@ -74,3 +74,91 @@ export const getPatientAge = (patient) => {
 
   return formatDuration(birthDuration, birthDuration.years > 5 ? { format: ['years'] } : { format: ['years', 'months'] });
 };
+
+export const getPatientAgeAtResourceDate = (resource, patientResource) => {
+  const birthDate = patientResource?.birthDate;
+  const resourceDate = format(resource.timelineDate, 'yyyy-MM-dd');
+  const ageAtResourceDate = intervalToDuration({
+    start: parse(birthDate, 'yyyy-MM-dd', new Date()),
+    end: parse(resourceDate, 'yyyy-MM-dd', new Date()),
+  });
```

Comment:
it seems that this would be re-computed, frequently...

Should this be injected similarly to `type`, `subtype`, and `timelineDate` ? 
(as a Date, but: formatted, here or in the component)

Reply:
also, is the 3rd arg, `baseDate`, used for some sort of fallback?

...either way: if the resource has no `timelineDate`,  it would have no `patientAgeAtResourceDate` (or whatever it would be called) -- or, should not have it?

...only things that have a `timelineDate` will show up in the timeline.

Reply:
Talked in IRL. Will move `getPatientAgeAtResourceDate` to be a field directly on the resource called `patientAgeAtTimelineDate`.